sys-apps/semodule-utils/semodule-utils-3.1.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+inherit toolchain-funcs
+
+MY_RELEASEDATE="20200710"
+SEPOL_VER="${PV}"
+SELNX_VER="${PV}"
+
+MY_P="${P//_/-}"
+IUSE=""
+
+if [[ ${PV} == *9999 ]] ; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/SELinuxProject/selinux.git"
+	S="${WORKDIR}/${MY_P}/${PN}"
+else
+	SRC_URI="https://github.com/SELinuxProject/selinux/releases/download/${MY_RELEASEDATE}/${MY_P}.tar.gz"
+	KEYWORDS="amd64 ~arm ~arm64 ~mips x86"
+	S="${WORKDIR}/${MY_P}"
+fi
+
+DESCRIPTION="SELinux policy module utilities"
+HOMEPAGE="https://github.com/SELinuxProject/selinux/wiki"
+
+LICENSE="GPL-2"
+SLOT="0"
+
+DEPEND=">=sys-libs/libsepol-${SEPOL_VER}:="
+
+# flatcar changes: add a weak blocker on policycoreutils-2.4
+# to prevent file collisions
+# policycoreutils-2.4 and semodule-utils provide the same files
+RDEPEND="${DEPEND}
+	!=sys-apps/policycoreutils-2.4-r2
+"
```

Comment:
If semodule-utils requires Flatcar changes, then why don't we move it to coreos-overlay?

Reply:
That's right - it's an edge case actually: starting from policycoreutils-2.7, `semodule-utils` becomes a single entity; see this commit: https://github.com/SELinuxProject/selinux/commit/c9c97d6e8bace621b8d91c1f5fb1e321f8d4c326.

In the current way we upgrade the things, `semodule-utils` needs to be installed **after** the `policycoreutils` upgrade - otherwise `semodule-utils-3.1` will collide with some files provided by `policycoreutils-2.4`.

This blocker allows portage to first upgrade `policycoreutils` then install `semodule-utils`.

We can certainly move `semodule-utils` to `::coreos-overlay` but we would need to move it back in `::portage-stable` once the upgrade done. :)